Makefile

```diff
@@ -110,6 +110,11 @@ get_vendor_deps:
 	@rm -rf .vendor-new
 	@dep ensure -v
 
+get_vendor_deps_ci:
```

Comment:
Almost an identical target to `get_vendor_deps` right? Why not just add a build env var/flag (i.e. `@dep ensure -v $(VENDOR)`) to the existing target?

Reply:
I'm not sure. I think makefiles tend to be super hard to read, so I prefer not using flags for the simple things in the main commands.

I don't feel strongly about this, so if you'd prefer flags id be happy to change it